js/start.js

```diff
@@ -54,6 +54,11 @@ app.localSettings.on('change:language', (localSettings, lang) => {
   .open();
 });
 
+app.hiRez = (
```

Comment:
1. the research I did said that you need to do both to catch all devices, so this makes it more reliable for someone running it on a tablet or something else weird. 

2. good idea.